saleor/dashboard/staff/emails.py

```diff
@@ -0,0 +1,20 @@
+from django.contrib.auth.tokens import default_token_generator
+from django.utils.encoding import force_bytes
+from django.utils.http import urlsafe_base64_encode
+
+from templated_email import send_templated_mail
+
+from ...settings import DEFAULT_FROM_EMAIL, get_host
+
+
+def send_set_password_email(staff):
+    ctx = {'protocol': 'http',
+           'domain': get_host(),
+           'site_name': 'saleor',
```

Comment:
We might want to retrieve site name from setting.

Reply:
Yes, thats true :)